www/scripts/productlist/ListOfProducts.js

```diff
@@ -213,15 +214,46 @@ function (declare, domClass, domConstruct, ItemFileWriteStore, topic,
         + name[0].toUpperCase()
         + '</span></div>';
 
-      if (!item.connected || !item.accessible) {
+
+      var dbStatus = item.databaseStatus
+
+      var db_status_text = ""
```

Comment:
This is a python style variable name :smile:. Use upper cased variable names in JavaScript.